test/utils/ermrest-import.js

```diff
@@ -43,6 +43,20 @@ exports.importSchemas = function(configFilePaths, catalogId) {
 	return defer.promise;
 };
 
+exports.importAcls = function(params) {
```

Comment:
I know it's obvious what this function is for, but can you add some comments to it. These utils functions are lacking documentation. Also the indentation in line 50 and 51 are off.

Reply:
They're just proxy functions. We already have documentation for them in ermrestDataUtils. Thats the reason they don't have any documentation.